### ISSUE TITLE
Tab Component: Improve Accessibility

### ISSUE THREAD

**Issue by mlmoravek** [oruga-ui/oruga#446]
### Description
Currently the tabs of the Tab Component can't reached by keyboard tabs by default. 

This can be solved by adding `tabindex="0"` to the TabsItem Component.   
Furthermore, the TabItem could also additionally get `role="button"` to get the HTML element appear as a button to screen readers.

### Why Oruga need this feature
Improve Accessibility

**Comment by jtommy** [comment]
@mlmoravek would you like to work on it?

**Comment by mlmoravek** [comment]
@jtommy I just had time to look at your implementation. You misunderstood the issue. You added the tabindex="0" to the div in the "nav-content" section element. But it should be on the "nav-item" div element to change the active tab by keyboard.

**Comment by astagi** [comment]
Thanks for your review @mlmoravek will work to improve a11y following your suggestions
